### ISSUE TITLE
Disabled text on Task Settings with dark theme is not legible

### ISSUE THREAD

**Issue by Scrxtchy** [ShareX/ShareX#4851]
![image](https://user-images.githubusercontent.com/12771982/83619449-3ee17e00-a5cf-11ea-9627-4529ffb6c0c8.png)

Text should be legible even if disabled. As it currently stands text to some users with impaired vision (but not impaired enough to rely on a screen reader) need to navigate around the UI to find the option to enable the respective settings to even understand if what they are looking for is the correct one at all

**Comment by Jaex** [comment]
Duplicate of #4813
